pkg/util/shell/testdata/data.txt

```diff
@@ -0,0 +1 @@
+Hello DevSpace!
```

Comment:
write temporary file before test instead of testdata

Reply:
done